news/wp-content/themes/cmubuggy-child/blocks/block-event.php

```diff
@@ -0,0 +1,16 @@
+<div class="card mb-3">
```

Comment:
Going to send a comment about this file to discord so others can weigh in.

Reply:
Agree these would be improvements – how would you feel about landing this as is to at least get the page updated and then I can come back and add these updates later this week?

Reply:
Yeah no problem, just wanted others to see it since this bit is more marketing and less pure tech :)